src/components/dialogs/parameters/load-flow-parameters.js

```diff
@@ -299,8 +299,8 @@ export const LoadFlowParameters = ({
     }, [resetParameters, resetProvider]);
 
     return (
-        <Grid container className={classes.grid}>
-            <Grid container key="provider">
+        <>
+            <Grid container className={classes.scrollableGrid} key="provider">
```

Comment:
This is not a problem of you PR. But you can change key="provider" to key="basicParameters" nope ?